acceptance/panic/output.txt

```diff
@@ -0,0 +1,13 @@
+
+>>> [CLI] panic
+The Databricks CLI unexpectedly panicked.
+Please report this issue to Databricks in the form of a GitHub issue at:
+https://github.com/databricks/cli
+
+CLI Version: [DEV_VERSION]
+
+Panic Payload: The databricks panic command always panics
```

Comment:
"Panic Payload" -> "Error message"?

Reply:
It could be anything though (via third party libraries). Not necessarily a string or an error.